app/services/ansible_fact_parser.rb

```diff
@@ -94,6 +94,10 @@ def kernel_version
     facts['ansible_kernel']
   end
 
+  def bios
+    facts['facter_dmi']['bios']
```

Comment:
I'd write this a bit safer. `facter_dmi` is only present if `facter` is present on the system so `facts['facter_dmi']` can be nil. In case it is `nil`, it should at least return a hash
```suggestion
    facts.dig('facter_dmi', 'bios') || {}
```